### PR TITLE
Fix write permissions for /home/kedro directory

### PR DESCRIPTION
**Pull request by DmitriiDeriabinQB**
## Motivation and Context
This is to fix permission issues when the container is run on some Linux hosts (an issue encountered by @lorenabalan when running on AWS Batch)

## Checklist

- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [x] Assigned myself to the PR
- [ ] Added tests to cover my changes
- [x] Added new entries to the `RELEASE.md` file

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: kedro_docker/plugin.py [hunk 1]
```diff
@@ -130,8 +130,8 @@ def docker_group():
             stderr=subprocess.DEVNULL,
             check=False,
         ).returncode
-    except FileNotFoundError:
-        raise KedroCliError(NO_DOCKER_MESSAGE)
+    except FileNotFoundError as file_not_found:
```
Comment (DmitriiDeriabinQB) [inline kedro_docker/plugin.py:133]: This is to make lint happy

File: features/steps/sh_run.py [hunk 2]
```diff
@@ -97,9 +97,7 @@ def __init__(self, cmd: Sequence[str], **kwargs) -> None:
             **kwargs: keyword arguments such as env and cwd
 
         """
-        super(ChildTerminatingPopen, self).__init__(
-            cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, **kwargs
-        )
+        super().__init__(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, **kwargs)
```
Comment (DmitriiDeriabinQB) [inline features/steps/sh_run.py:100]: This is to make lint happy

**Review by limdauto — APPROVED** [review]
LGTM

**Review by lorenabalan — APPROVED** [review]
Thanks for addressing so quickly. Just one cosmetic change, looks good otherwise. 🏆

File: kedro_docker/plugin.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (DmitriiDeriabinQB) [reply kedro_docker/plugin.py:133]: I've copied it from #40 but I agree, let me update 🙂

**Pull request merged**